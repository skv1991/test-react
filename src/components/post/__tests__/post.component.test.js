import React from 'react';
import Post from '../component';
import {shallow} from "enzyme";

describe('Post', () => {
  it('Рендеринг', () => {
    expect(shallow(<Post title="Заголовок" text="Текст" />)).toMatchSnapshot();
  });
});
