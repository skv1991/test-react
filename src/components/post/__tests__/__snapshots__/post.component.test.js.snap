// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post Рендеринг 1`] = `
<article
  className="post"
>
  <h1
    className="post__title"
  >
    Заголовок
  </h1>
  <div
    className="post__text"
  >
    Текст
  </div>
</article>
`;
